torch/nested/_internal/ops.py

```diff
@@ -280,20 +280,6 @@ def jagged_torch_function(func, *args, **kwargs):
     if func is torch._C._nn.scaled_dot_product_attention:
         return jagged_scaled_dot_product_attention(*args, **kwargs)
 
-    # Handle reshape() / reshape_as() here because they're CompositeImplicit.
```

Comment:
NB: Handling these in `__torch_function__()` this isn't enough to get them working for autograd formulas.